examples/embedding/main.go

```diff
@@ -48,14 +50,24 @@ func main() {
 
 	// add documents
 	err = chroma.AddDocuments(context.TODO(), []schema.Document{
-		{PageContent: "This is a document about cats. Cats are great."},
```

Comment:
@Lanture1064 Here you removed this document. Why?

Reply:
It is not removed, but seperated into 2 `AddDocuments` commands.